test/test_files.py

```diff
@@ -0,0 +1,64 @@
+from files import *
+
+
+def test_files():
+    in_file = "./samples/xml/test_data.xml"
+    out_file = "./samples/json/updated_test_data.json"
+
+    print("\n\n=== Start 'test_file' test ===")
+
+    print("\n\nStart 'test_file' test")
+
+    print("\n\nClean-up")
+    clean_up(out_file)
+
+    print("\n\nCheck if file source XML file exists\nFile: ", in_file)
+    result = is_file_exist(in_file)
+    if result:
```

Comment:
Using if in tests is a bad practice. Please, convert if statement to a separate test.


```
https://stackoverflow.com/questions/15154125/if-statements-in-tests

In my opinion a Junit should be read like a protocol. That means you can write redundant code to make the test case better readable. Write a testcase for each possible if-statement in your business logic as well as the negative cases. Thats the only way to get a 100% test coverage. I use the structure:

- testdata preparation
- executing logic
- check results
- clear data
```